templates/auth/principals/operators.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.identity.enabled }}
```

Comment:
I think these principal entries should be enabled based on the individual services that need them, rather than all being enabled with `identity`. Otherwise we can't migrate individual services across.

Reply:
Agreed, good spot.